components/header/Buttons/Language.tsx

```diff
@@ -0,0 +1,35 @@
+import Icon from "deco-sites/bolovo-store/components/ui/Icon.tsx";
+import type { ImageWidget } from "apps/admin/widgets.ts";
+import Image from "apps/website/components/Image.tsx";
+import { usePartialSection } from "deco/hooks/usePartialSection.ts";
+
+export default function Lenguage({ countryFlag }: {
+    countryFlag: { countryLabel: string, countryImg: { img: ImageWidget, alt: string } }[];
+}) {
+    return (
+        <>
+            <details className="dropdown">
+                <summary className=" btn px-0 flex flex-row flex-nowrap border-none hover:border-none bg-transparent hover:bg-transparent w-max   text-base font-normal" >
+                    <Image src={countryFlag[0].countryImg.img} alt={countryFlag[0].countryImg.alt} width={19} height={12} />
+                    {countryFlag[0].countryLabel}
+                    <Icon
+                        id="ChevronDown"
+                        size={11}
+                        strokeWidth={2}
+                        fill="none"
+                    /></summary>
+                <ul className="p-2 shadow menu w-auto dropdown-content z-[1] bg-base-100 rounded-box">
+                    {countryFlag.map((iten, index) => {
+                        return (
+                            <button className=" cursor-pointer m-1 flex justify-center items-center uppercase gap-2 h-8 px-0 flex-row flex-nowrap border-none hover:border-none bg-transparent hover:bg-transparent w-max   text-base font-normal"
+                                {...usePartialSection({ props: { itenIndex: index } })}>
```

Comment:
erro de escrita novamente, deveria ser `itemIndex`. Mas nao entendi o que é isso... Na verdade voce deveria estar passando aqui qual a linguagem, tipo "BR", "EN" e etc, entenda que isso deve ser recebido nas props do Header (e usado posteriormente em algum loader inline ou coisa do tipo)

```suggestion
                                {...usePartialSection({ props: { itemIndex: index } })}>
```